frontend/src/stimulus/controllers/dynamic/job-status-polling.controller.ts

```diff
@@ -40,14 +40,18 @@ export default class JobStatusPollingController extends Controller<HTMLElement>
     setTimeout(() => element.click(), 50);
   }
 
-  redirectClick(_:Event) {
-    this.userInteraction = true;
+  redirectClick(event:PointerEvent) {
+    this.followLink(event.target as HTMLLinkElement);
```

Comment:
Before the change the actual navigating was done by the browser. 
As we now set windows.href ourselves, i think it's best to ensure stopping the browser one via the event.

```suggestion
    event.preventDefault();
    this.followLink(event.target as HTMLLinkElement);
```

Reply:
Thanks, I committed your suggestion.

I think the behavior when clicking is intercepted by turbo-something, and it tries to replace the turbo-frames from the current page with the turbo-frames from the response, which in turn totally skips the redirection which normally happens. Not entirely sure though.